tests/settings.py

```diff
@@ -6,30 +6,48 @@
 django_stubs_ext.monkeypatch()
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+DEBUG = True
```

Comment:
Question: What is `DEBUG` needed for? This changes the semantics of the tests a little, so we shouldn't do it unless we **have** to.

Perhaps this too can be moved to a separate settings file along with the `TASKS` setting to be used when someone is running the test app directly, vs using it for the test suite.

Reply:
We need debug for static files in django